polars/lazy/dataframe.ts

```diff
@@ -176,7 +176,38 @@ export interface LazyDataFrame extends Serialize, GroupByOps<LazyGroupBy> {
    */
   head(length?: number): LazyDataFrame;
   /**
-   * Add a join operation to the Logical Plan.
+   *  __SQL like joins.__
```

Comment:
we should document the `allowParallel` and `forceParallel` options as well.

Reply:
Good point, added these